tests/scale/README.md

```diff
@@ -0,0 +1,154 @@
+			Dashboard: "ABCDEF/mesh-cpu-and-mem",
+			Panel: 14,
+		},
+		{
+			Filename: "mem",
+			Dashboard: "ABCDEF/mesh-cpu-and-mem",
+			Panel: 12,
+		},
+	}
+}
+```
+
+## Results
+ Iteration results will display on test logs to keep an idea of progress.
+Only observed resources at this point in time are displayed.
+```
+-- Successfully completed iteration 8 - took 1m36.700005257s
++------------------------+--------------+---------+
+|        Resource        | CpuAvg (96s) | Mem RSS |
++------------------------+--------------+---------+
+| osm-cont..p2rb/osm-c.. |         0.44 | 160 MB  |
+| osm-prom..n8gt/prome.. |         0.19 | 1.2 GB  |
++------------------------+--------------+---------+
+```
+
+Upon `WrapUp` call time, a more comprehensive view of the evolution of resources is displayed at the end of the test, displaying both CPU avg for iteration and RSS footprint at the end, as well as relative increase to previous iteration.
+
+Note that this will show all possible resources that ever appeared during the test that matched any tracked label (so in case of restarted/crashed pods, all metrics and iterations present will also show):
+```[AfterEach] ScaleClientServerTrafficSplit
+  /home/eserra/src/osm/tests/scale/scale_trafficSplit_test.go:22
++----+---------------------------+-------+-------------------------+-------------------------+
+| It |         Duration          | NPods | osm-prom..n8gt/prome..  | osm-cont..p2rb/osm-c..  |
++----+---------------------------+-------+-------------------------+-------------------------+
+|  0 | 32.593299891s             |    20 | err / 126 MB            | 0.03 / 46 MB            |
+|  1 | 32.54950816s (-0.13%)     |    40 | err / 272 MB (+115.60%) | err / 55 MB (+19.84%)   |
+|  2 | 32.608644378s (+0.18%)    |    60 | 0.13 / 376 MB (+38.23%) | err / 66 MB (+20.46%)   |
+|  3 | 34.7493147s (+6.56%)      |    80 | 0.14 / 535 MB (+42.28%) | 0.55 / 82 MB (+24.04%)  |
+|  4 | 40.077845174s (+15.33%)   |   100 | 0.15 / 625 MB (+16.91%) | 0.66 / 96 MB (+16.16%)  |
+|  5 | 38.266130228s (-4.52%)    |   120 | 0.13 / 725 MB (+16.00%) | 0.59 / 102 MB (+7.16%)  |
+|  6 | 49.027804296s (+28.12%)   |   140 | 0.20 / 883 MB (+21.72%) | 0.94 / 124 MB (+21.51%) |
+|  7 | 56.293969628s (+14.82%)   |   160 | 0.19 / 1.0 GB (+15.16%) | 1.37 / 140 MB (+12.44%) |
+|  8 | 1m36.700005257s (+71.78%) |   180 | 0.21 / 1.2 GB (+17.39%) | 0.44 / 160 MB (+14.70%) |
++----+---------------------------+-------+-------------------------+-------------------------+
+```
+A per-pod RSS footprint average increase per pod is also provided for the runtime of the test:
+```
++------------------------+-----------+
+|        Resource        | Mem / pod |
++------------------------+-----------+
+| osm-prom..n8gt/prome.. | 6.7 MB    |
+| osm-cont..p2rb/osm-c.. | 715 kB    |
++------------------------+-----------+
+```
+
+Additionally, if Grafana rendering was present and enabled, any added dashboards will be saved in a test folder.
```

Comment:
> any added dashboards will be saved in a test folder.

If this is stored in the repo directory by default, we should probably add an entry in `.gitignore` just to keep them from getting checked in.

Reply:
true

Reply:
done